docs.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+MODIFIED_FILES=$(git --no-pager diff --name-only ${TRAVIS_COMMIT_RANGE})
+if grep -q "docs/" <<<$MODIFIED_FILES
+then
+  sudo apt-get install -y aspell-en
+  modified_files_arr=($MODIFIED_FILES)
+  for docs_file_modified in "${modified_files_arr[@]}"
+  do
+    if grep -q "docs/" <<<$docs_file_modified
+    then
+      <$docs_file_modified aspell pipe list -d en_US --encoding utf-8 --personal=./.aspell.en.pws |
+      grep '[a-zA-Z]\+ [0-9]\+ [0-9]\+' -oh |
+      grep '[a-zA-Z]\+' -o | sort | uniq |
+      while read word; do
+        grep -on "\<$word\>" $docs_file_modified;
+      done >>./misspelled_per_file.txt;
+      if [[ -s ./misspelled_per_file.txt ]]
+      then
+        sort -n ./misspelled_per_file.txt -o ./misspelled_per_file.txt;
+        echo "Misspelled words in File : $docs_file_modified :- " > ./Misspelled_words.txt;
+        cat ./misspelled_per_file.txt >> ./Misspelled_words.txt;
+      fi
+    fi
+  done
+  if [[ -s ./Misspelled_words.txt ]]
+  then
+    echo "~~~  Spelling Errors  ~~~ "
+    cat ./Misspelled_words.txt
+    return 1;
+  fi
+  rm -rf ./misspelled_word_per_file.txt
+  rm -rf ./Misspelled_words.txt
+  mkdir out;
+  cd out
+  git clone https://github.com/ParsaLab/cloudsuite.git
```

Comment:
All these git commands should only get executed if we want to merge.
And merge only happens in case of a push to master (not a pull request).